doc/build_apps/example_rpc_api.rst

```diff
@@ -6,4 +6,5 @@ Example app RPC API
    :include:
       /app/log/.*
       /app/api
```

Comment:
/app/api is also built-in, effectively

Reply:
Ahh okay, in which case I think we should include `/app/api` in `use_apps/rpc_api.rst` and only expand `api/log/` here